lib/mongoid/errors/invalid_storage_parent.rb

```diff
@@ -1,25 +0,0 @@
-# frozen_string_literal: true
-
-module Mongoid
-  module Errors
-
-    # Raised when calling store_in in a sub-class of Mongoid::Document
-    class InvalidStorageParent < MongoidError
```

Comment:
The exception class needs to be retained if this change is going into 8.1. It should be marked deprecated and won't be used by Mongoid.